src/core/lib/iomgr/executor/threadpool.h

```diff
@@ -62,7 +62,7 @@ class ThreadPoolInterface {
 // NULL closure.
 class ThreadPoolWorker {
  public:
-  ThreadPoolWorker(const char* thd_name, ThreadPoolInterface* pool,
+  ThreadPoolWorker(const char* thd_name, ThreadPoolInterface* /*pool*/,
```

Comment:
I think we forgot to remove this parameter. It is unnecessary in ThreadPoolWorker. I will remove it in a later PR.